js_modules/dagster-ui/packages/ui-core/src/search/useGlobalSearch.tsx

```diff
@@ -561,49 +505,3 @@ export const SEARCH_PRIMARY_QUERY = gql`
 
   ${PYTHON_ERROR_FRAGMENT}
 `;
-
-export const SEARCH_SECONDARY_QUERY = gql`
-  query SearchSecondaryQuery {
-    assetsOrError {
-      ... on AssetConnection {
-        nodes {
-          id
-          ...SearchAssetFragment
-        }
-      }
-    }
-  }
-
-  fragment SearchAssetFragment on Asset {
-    id
-    key {
-      path
-    }
-    definition {
-      id
-      computeKind
-      groupName
-      owners {
-        ... on TeamAssetOwner {
-          team
-        }
-        ... on UserAssetOwner {
-          email
-        }
-      }
-      tags {
-        key
-        value
-      }
-      kinds
-      repository {
-        id
-        name
-        location {
-          id
-          name
-        }
-      }
-    }
-  }
-`;
```

Comment:
you ensured this lines up with whats available on the workspace context?

Reply:
Yeah we use AssetTableDefinitionFragment in the WorkspaceQuery https://github.com/dagster-io/dagster/blob/c75493ece7989e6d40aef240a3e182f23cce211f/js_modules/dagster-ui/packages/ui-core/src/assets/AssetTableFragment.tsx#L4-L61